src/main/logutils.ts

```diff
@@ -177,9 +337,9 @@ const handleArenaStartLine = (line: string) => {
     // Helpfully ARENA_MATCH_END events contain the game duration. Solo shuffle
     // ARENA_MATCH_END duration only counts the last game so needs special handling. 
     if (metadata.category !== "Solo Shuffle") {
-        duration = parseInt(line.split(',')[2]);
+        duration = parseInt(line.args[2], 10);
```

Comment:
We chatted about this 10 arg already. Is it technically possible to redefine parseInt? e.g.

```
const parseInt (i) => {
  return parseInt(i, 10);
}
```

Reply:
If not, whatever - not a big deal.

Reply:
No, you can't redefine it, but we can literally just make a "parseInt10()" or something if you prefer a kind of syntactic sugar of sorts - shorthand or whatever.

Reply:
It's also possible to _remove_ the `radix` parameter and let JS sort it out - it _did_ work before and would probably cause no problems whatsoever to _not_ specify it.

Reply:
_Alternatively_, I could create methods on `LogLine` like:

```typescript
arg(index: number): any;
intArg(index: number): number;
hexArg(index: number): number;
```

That would then facilitate this and wrap it in those. That wouldn't take even a minute to fix.

Reply:
Eh, not going to lose sleep over this then. Fine as is.

Reply:
Happy to leave as per other comment.